spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -82,6 +110,21 @@ def _handle_input_request(self, msg):
         # Make sure that all output from the SUB channel has been processed
         # before entering readline mode.
         self.kernel_client.iopub_channel.flush()
+        self._input_ready = True
+        if not self.is_debugging():
+            return super(DebuggingWidget, self)._handle_input_request(msg)
+
+        # While the widget thinks only one input is going on,
+        # Other functions are sending messages to the kernel.
```

Comment:
Other -> other

Reply:
Also is right to says

    other functions are sending messages to the kernel

or would it be better to say

    other functions can be sending messages to the kernel